gzip.go

```diff
@@ -71,6 +75,12 @@ type GzipResponseWriter struct {
 
 // Write appends data to the gzip writer.
 func (w *GzipResponseWriter) Write(b []byte) (int, error) {
+	// If the response is not bigger than the defined minimum size
+	// before compression. The regular response writer is called instead
```

Comment:
This comment doesn't say anything which the code doesn't already. Let's remove it.